tools/tests/bundle-ignore-files.js

```diff
@@ -12,4 +12,10 @@ selftest.define("bundle-ignore-files", () => {
   ];
 
   matchingInputs.forEach(input => selftest.expectEqual(patterns.some(p => p.test(input)),true));
+
+  const nonMatchingInputs = [
+    '/imports/components/Icon/index.js',
```

Comment:
This should actually just be `Icon` (this only matches the segment, not the full path).

Can you add `Icon.` to the `matchingInputs` test too?  Truthfully, I have no idea what an `Icon.` file is in the first place and the original commit (63092aabc4067a505ae1c1269e03f7a2b67b68c0) didn't mention, but it seems to be some sort of cache file that should be ignored and we might as well keep with that.